app/assets/stylesheets/atoms/_form-elements.scss

```diff
@@ -14,6 +14,34 @@
   }
 }
 
+.checkbox--simplified {
```

Comment:
this whole thing feels like it looks questionable to me but between:
a) using both the old class `.checkbox` and a new class `.checkbox--simplified` and overwriting old styles with the new class (current implementation)
and 
b) "starting from scratch" using just `.checkbox--simplified` and trying to copy the needed styles over from `.checkbox`
after playing around a bit, (a) seemed much simpler

Reply:
did you try using the hub_checkbox form helper we talked about the other day? I think we were going to try renaming it something like simple_checkbox or something.